alias/README.md

```diff
@@ -0,0 +1,35 @@
+# Git kata: Aliases
+
+While working, we tend to write a lot of commands. This can get tedious, especially when the commands are rather long. Git allows you to create an alias for commands, dramatically shortening the time spent typing them.
+
+Aliases are stored in your git config and can thus be global or local. Global config is tied to your user, while local config lives inside a specific repository.
```

Comment:
They might also be system? System, User, local :D I know we mostly work with global/local, but still :D

Reply:
Good point. I'll clarify this.